projects/go-lib/src/lib/services/go-message/go-message.module.ts

```diff
@@ -5,7 +5,9 @@ import { GoMessageService } from './go-message.service';
 
 @NgModule({
   declarations: [],
-  providers: [GoMessageService],
+  providers: [
+    GoMessageService
```

Comment:
Do we still need this if the `GoMessageService` is `providedIn: 'root'`?

Reply:
This is actually going to be deprecated by the ToasterService so I'm not too worried about it.